anni-playback/src/cpal_output.rs

```diff
@@ -159,7 +159,7 @@ impl CpalOutputStream {
         }
     }
 
-    fn get_config(_spec: SignalSpec) -> anyhow::Result<(Device, StreamConfig)> {
+    fn get_config(spec: SignalSpec) -> anyhow::Result<(Device, StreamConfig)> {
```

Comment:
It may produce a warning on windows? Maybe we can add `let _ = spec;` in windows branch.

Reply:
Or using cfg_attr?
```diff
diff --git a/anni-playback/src/cpal_output.rs b/anni-playback/src/cpal_output.rs
index d54116c..1519847 100644
--- a/anni-playback/src/cpal_output.rs
+++ b/anni-playback/src/cpal_output.rs
@@ -159,7 +159,9 @@ impl CpalOutputStream {
         }
     }

-    fn get_config(_spec: SignalSpec) -> anyhow::Result<(Device, StreamConfig)> {
+    fn get_config(
+        #[cfg_attr(target_os = "windows", allow(unused))] spec: SignalSpec,
+    ) -> anyhow::Result<(Device, StreamConfig)> {
         let host = cpal::default_host();
         let device = host
             .default_output_device()
```

Reply:
LTGM